website/.prettierignore

```diff
@@ -14,4 +14,5 @@ ember-cli-update.json
 *.scss
 
 # temporary ignore
-/docs/
+/docs/**/*.md
```

Comment:
I made the prettier ignore more specific so the code snippets would be fixed by prettier.